va/va.go

```diff
@@ -668,6 +678,18 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, req *v
 			logEvent.Challenge.Status = core.StatusValid
 		}
 
+		if va.perspective != "" && va.perspective != PrimaryPerspective {
+			// This validation was performed by a remote VA. According to the
+			// requirements in section 5.4.1 (2) vii of the BRs we need to log
+			// the perspective used. Additionally, we'll log the RIR where this
+			// RVA is located.
+			//
+			// TODO(#7615): Make these fields mandatory for non-Primary
+			// perspectives and remove the (va.perspective != "") check.
+			logEvent.Perspective = va.perspective
+			logEvent.RIR = va.rir
+		}
```

Comment:
nit: couldn't we just set both of these on the logEvent regardless? Including "perspective: Primary" in the local logs doesn't seem like a terrible thing, and the RIR will be left out because it's omitempty. Certainly no need to change this, but I think it would be fine for this to be as simple as possible.

Reply:
I agree, I'll take care of this in the next phase.